frontend/src/Judge/components/ShowCard.js

```diff
@@ -18,39 +19,93 @@ const ButtonContainer = styled.div`
   bottom: 0;
 `
 
-const ShowCard = props => (
-  <Card>
-    <h2>
-      <Link to={`show/${props.id}`}>{props.name}</Link>
-    </h2>
+function renderCardContents (props) {
```

Comment:
```javascript
const CardContents = props => {
```

Reply:
I would just get rid of this function. See my comment on your comment below.